tests/system/global.classic.STEHE.restart.FROZEN_SOIL.txt

```diff
@@ -0,0 +1,101 @@
+#######################################################################
+# VIC Model Parameters for Stehekin Basin Sample Classic Driver Setup
+#######################################################################
+
+#######################################################################
+# Simulation Parameters
+#######################################################################
```

Comment:
Comment for all the new global parameter files required for testing. Please remove all comments unless specific to this particular configuration.  I don't like maintaining the comments in these files in 10 places so its easier to just specify the option you are using.  See the examples tests for a good *example*.